Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7-slim AS base
+
+FROM base AS build
+RUN apt-get update && \
+    apt-get install -y \
+    build-essential \
+    libyaml-dev
+COPY requirements.txt /
+RUN pip install --no-cache-dir -r /requirements.txt
+RUN pip install py-bcrypt
```

Comment:
We don't really provide the tools to run manhole automatically normally. Should we here?

Reply:
I think it's fine. The size difference in the docker image is negligible. Besides that it would be really annoying to install later since you'd have to build a new image.

Reply:
Fair point. I put this line in comments and added an explanation.